Jenkinsfile_template

```diff
@@ -0,0 +1,32 @@
+#!groovy
+properties(
+    [
+        [
+            $class       : 'GithubProjectProperty',
+            projectUrlStr: 'https://github.com/hmcts/spring-boot-template'
+        ],
+        pipelineTriggers([[$class: 'GitHubPushTrigger']])
+    ]
+)
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "rpe"
+def component = "demo"
+
+
+withPipeline(type, product, component) {
+    installCharts()
+    disableLegacyDeployment()
+
+    onMaster {
+        enableSlackNotifications('#rpe-spring-demo-builds')
+    }
+    onDemo {
+        enableSlackNotifications('#rpe-spring-demo-builds')
+    }
+    onPR {
+        enableSlackNotifications('#rpe-spring-demo-builds')
+    }
```

Comment:
I would recommend against this as mentioned in https://github.com/hmcts/spring-boot-template/pull/187/files#r427355521
```suggestion
```

Reply:
done